content/asset-tracking/using-the-sdks.textile

```diff
@@ -600,8 +601,82 @@ subscriber.nextLocationUpdateIntervals
 
 See also the SDK reference docs for more "Subscriber properties":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/subscribing-sdk/subscribing-sdk/com.ably.tracking.subscriber/-subscriber/index.html.
 
+h2(#location-animator). Location animator
+
+The location animator is an extension for the subscriber that provides smooth location updates animation. 
+
+The location animator receives location updates from the publisher and creates animation steps from them. These steps are then applied to an ongoing animation. 
+
+The animation duration is then recalculated as the sum of location update interval (the time to next location update from the publisher) and the intentional delay. The intentional delay allows for network issues or delays in the location updates. For example, if the intentional delay was set to two seconds, then a two second buffer will help the animation remain smooth and continuous, even if the publisher did not send a location update within the specified time, with a two second margin.
+
+The location animator then recalculates each step duration to evenly take up the animation duration, so the overall animation of the trackable is smooth.
+
+Each step is animated at 60 FPS, and this applies to both the map marker and the camera position.
```

Comment:
"this applies to [...] the camera position." - not true 😅 

The camera positions are not animated on their own, we simply publish a single camera position whenever it needs to be updated. We expect the users to use their map provider SDK to animate the camera from the current position to the new one. Otherwise, the camera would not be responsive for user input if we were to update its position 60 times per second 😉 
```suggestion
Each step is animated at 60 FPS, and this applies to the map marker.
```